README.md

```diff
@@ -31,7 +31,7 @@ It is a general purpose statically typed, easy to learn and use language.
 - strong static typing
 
 #### Manuals:
-- [User Manual](https://jsmith1024.github.io/bill/html/User.html)
-- [Reference Manual](https://jsmith1024.github.io/bill/html/Reference.html)
-- [Bill Primer](https://jsmith1024.github.io/bill/html/Primer.html)
-- [Standard Library Reference](https://jsmith1024.github.io/bill/html/Standard.html)
+- Bill Primer [HTML](https://jsmith1024.github.io/bill/html/Primer.html) [PDF](https://jsmith1024.github.io/bill/Primer.pdf)
+- Reference Manual [HTML](https://jsmith1024.github.io/bill/html/Reference.html) [PDF](https://jsmith1024.github.io/bill/Reference_Manual.pdf)
+- Standard Library Reference [HTML](https://jsmith1024.github.io/bill/html/Standard_Library_Reference.pdf) [PDF](https://jsmith1024.github.io/bill/Library.html)
```

Comment:
Addresses are swapped.

Reply:
I believe that was already fixed in another commit.  Note the Outdated next to the filename.